arkane/qchem.py

```diff
@@ -273,12 +273,13 @@ def loadEnergy(self, frequencyScaleFactor=1.):
         """
         E0 = None
         with open(self.path, 'r') as f:
-            for line in f:
+            lines = f.readlines()
+            for line in lines:
                 if 'Final energy is' in line:
                     E0 = float(line.split()[3]) * constants.E_h * constants.Na
                     logging.debug('energy is {}'.format(str(E0)))
             if E0 is None:
-                for line in f:
```

Comment:
I like your implementation. I'll modify it a bit into
`e0 = a if a else b`
since I think that if the text for b is in the file, then the text for a is there as well. I'll check and will anyway add a test.